lib/std/os/windows.zig

```diff
@@ -952,7 +955,53 @@ pub fn SetFilePointerEx_CURRENT_get(handle: HANDLE) SetFilePointerError!u64 {
     return @bitCast(u64, result);
 }
 
+pub fn QueryObjectName(
+    handle: HANDLE,
+    out_buffer: []u16,
+) ![]u16 {
+    var full_buffer: [@sizeOf(OBJECT_NAME_INFORMATION) + PATH_MAX_WIDE * 2]u8 align(@alignOf(OBJECT_NAME_INFORMATION)) = undefined;
```

Comment:
Why 2 buffers with a memcpy? We could pass out_buffer directly to NtQueryObject right?

Reply:
Yes, that's what the code did originally, which was commented on [here](https://github.com/ziglang/zig/pull/7379#discussion_r540323038), so I changed it.
I can change it back if you want - I don't think there was a particular reason besides readability - but I don't like stepping on anyone's toes, so I'll put that in as the last commit and then we can pick whichever turns out looking nicer.